config/common/common.go

```diff
@@ -41,6 +41,15 @@ const (
 	nameReferenceKind = "name"
 	// idReferenceKind represents ID reference kind
 	idReferenceKind = "id"
+
+	// VersionV1Alpha2 is used as minimum version for all manually configured resources.
+	VersionV1Alpha2 = "v1alpha2"
```

Comment:
Could you please make sure that it's okay `v1alpha1` versions of existing resources? Normally, we would have to have `v1alpha1` and `v1alpha2` versions simultaneously supported for upgrades and `v1alpha1`s removed at a later stage (e.g., at the end of deprecation period). However, I don't think upgrades are a consideration atm.